spec/features/feedback_spec.rb

```diff
@@ -32,4 +32,14 @@
     expect(page).to have_css 'h1', text: 'Feedback submitted'
     expect(page).not_to have_css 'a', text: 'Return to page'
   end
+
+  scenario 'feedback banner is not shown on feedback page' do
+    visit '/404'
+    expect(page).to have_css 'a', text: 'feedback'
+    expect(page).to have_css 'p', text: 'Tell us what you think - your feedback will help us improve.'
+
+    click_on 'feedback'
```

Comment:
What happens when we click on the button?
does is show something like "thank you for your feedback" :thinking:  
not sure, but I would expect that.
In that case, we should test it.

Reply:
that's already tested in it's own feature spec, which I merged in the previous PR, This PR is is just a link and it's testing the link and showing you have reached the correct page